### PR TITLE
Added convenience method to set properties for all MessageSizeCalcula…

### PR DESCRIPTION
**Pull request by zhongwuzw**
Support for #694 .

### CODE REVIEW

**Comment by zhongwuzw** [comment]
@SD10 Let's provide one shot method.

File: Sources/Layout/MessagesCollectionViewFlowLayout.swift [hunk 1]
```diff
@@ -155,4 +155,70 @@ open class MessagesCollectionViewFlowLayout: UICollectionViewFlowLayout {
         let calculator = cellSizeCalculatorForItem(at: indexPath)
         return calculator.sizeForItem(at: indexPath)
     }
+    
+    /** Set `incomingAvatarSize` of all `MessageSizeCalculator`s */
```
Comment (BasThomas) [inline Sources/Layout/MessagesCollectionViewFlowLayout.swift]: Nit, but what about a triple `/` (`///`) instead of these multiline comments?

**Comment by ndonald2** [comment]
Looks good to me! Thanks for doing this.

File: Sources/Layout/MessagesCollectionViewFlowLayout.swift [hunk 2]
```diff
@@ -155,4 +155,70 @@ open class MessagesCollectionViewFlowLayout: UICollectionViewFlowLayout {
         let calculator = cellSizeCalculatorForItem(at: indexPath)
         return calculator.sizeForItem(at: indexPath)
     }
+    
+    /// Set `incomingAvatarSize` of all `MessageSizeCalculator`s
+    open func setMessageIncomingAvatarSize(_ newSize: CGSize) {
```
Comment (SD10) [inline Sources/Layout/MessagesCollectionViewFlowLayout.swift]: Think these should be `public` and not `open`? They're helper methods. I don't know if we should allow people to override? I don't see any major harm in leaving it but what do you think?

**Comment by SD10** [comment]
🙏 I'm so lucky to have other contributors 😭

File: Sources/Layout/MessagesCollectionViewFlowLayout.swift (hunk 2 above)
Comment (zhongwuzw) [reply Sources/Layout/MessagesCollectionViewFlowLayout.swift]: No big differences I think, I'll change this to `public`.

**Review by SD10 — APPROVED** [review]
Great 💯

**Pull request merged**